apps/src/p5lab/P5Lab.js

```diff
@@ -254,7 +254,8 @@ export default class P5Lab {
     config.usesAssets = true;
 
     this.studioApp_.labUserId = config.labUserId;
-    this.studioApp_.storage = initFirebaseStorage({
+    // TODO: unfirebase, convert to datablock storage: #56995
+    this.studioApp_.storage = initStorage(FIREBASE_STORAGE, {
```

Comment:
For now we are only enabling DatablockStorage for Applab. Changing this line to be like applab (https://github.com/code-dot-org/code-dot-org/blob/384f3f51cf53b9b820a18fbd3f1a503051f0d3b3/apps/src/applab/applab.js/#L421-L443) should be all that's required to switch gamelab over too.